demisto_sdk/main.py

```diff
@@ -468,6 +471,35 @@ def init(**kwargs):
     return 0
 
 
+# ====================== generate-docs ====================== #
+@main.command(name="generate-docs",
+              short_help="Generate documentation for integration, playbook or script from yaml file.")
+@click.help_option(
+    '-h', '--help'
+)
+@click.option(
+    "-i", "--input", help="path of the yml file.", required=True)
+@click.option(
+    "-o", "--output", help="The output dir to write the documentation file into.", required=True)
+@click.option(
+    "-t", "--file_type", type=click.Choice(["integration", "script", "playbook"]),
+    help="The type of yml file.", required=True)
+@click.option(
+    "-c", "--commands", help="Path for file containing command examples. Each Command should be in a separate line."
+                             " For example: !fidelis-list-alerts time_frame=Last 7 Days, required=False")
+@click.option(
+    "-id", "--id_set", help="Path of updated id_set.json file.", required=False)
+@click.option(
+    "-v", "--verbose", is_flag=True, help="Verbose output - mainly for debugging purposes")
+def generate_doc(file_type, **kwargs):
+    if file_type == 'integration':
+        return generate_integration_doc(**kwargs)
+    elif file_type == 'script':
+        return generate_script_doc(**kwargs)
+    elif file_type == 'playbook':
+        return generate_playbook_doc(**kwargs)
```

Comment:
```
else:
    print_error(...)
    return 1
```

Reply:
BTW if the type is not the the list of the options, you gets this error:
`Error: Invalid value for "-t" / "--file_type": invalid choice: widget. (choose from integration, script, playbook)
`